src/locale/messages.da.xlf

```diff
@@ -0,0 +1,187 @@
+        <target>Tjek din forbindelse og om du har konfigureret Ingest Gateway.</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown-overview/components/rundown-overview/rundown-overview.component.html</context>
+          <context context-type="linenumber">12</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="172210689759305381" datatype="html">
+        <source>rundown-overview-component.rundown-label</source>
+        <target>Rundown</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown-overview/components/rundown-overview/rundown-overview.component.html</context>
+          <context context-type="linenumber">26</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="4359206169774515973" datatype="html">
+        <source>rundown-overview-component.last-updated.label</source>
+        <target>Senest opdateret:</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown-overview/components/rundown-overview/rundown-overview.component.html</context>
+          <context context-type="linenumber">33</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="690878947555894079" datatype="html">
+        <source>rundown-header-context-menu-component.reset-rundown.button</source>
+        <target>Nulstille</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.html</context>
+          <context context-type="linenumber">4</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="221016838323992979" datatype="html">
+        <source>rundown-header-context-menu-component.activate.button</source>
+        <target>Aktivere</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.html</context>
+          <context context-type="linenumber">8</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="2175030815390487298" datatype="html">
+        <source>rundown-header-context-menu-component.deactivate.button</source>
+        <target>Deaktivere</target>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/rundown/components/rundown-header-context-menu/rundown-header-context-menu.component.html</context>
+          <context context-type="linenumber">12</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="1013806782562403538" datatype="html">
+        <source>rundown-header-context-menu-component.take.button</source>
+        <target>Take</target>
```

Comment:
Are we also calling it a "Take" in danish?

Reply:
Perhabs it should be "Tag"? 
Sounds weird, but should be the direct translation.

Reply:
I think we should ask Jacob what he prefers

Reply:
I think the best is to leave it as "take"